modules/volunteers/admin/sitesupport/editHours.js

```diff
@@ -0,0 +1,116 @@
+/* jshint esversion: 6 */
+/* globals apiRequest, confirmbox, showSidebar, simpleObjectToRequest */
+
+export default {
+  data() {
+    return {
+      record: {
+        hours: 0,
+        member: {
+          first_name: '',
+        },
+        enterer: {
+          id: '',
+        },
+        authorizer: {
+          id: '',
+        },
+        department: {
+          name: '',
+        },
+        end: null,
+      },
+    }
+  },
+  methods: {
+    commitHours() {
+      confirmbox(
+        'Please! double check entries',
+        'Proceed with Volunteer Hour Update?').then(() => {
+        var item = {};
```

Comment:
Another option for filling in object details would be like this:

```
const item = {
  hours: parseFloat(this.record.hours),
  end: this.record.end,
  // ignoring modifier for now until I have a chance to look at binding with `<option>` tags
  department: this.record.department.id,
  authorizer: this.record.authorizer.id
};
```

There's not much of a difference in how we decide to do it, but it can get a bit cleaner when it comes to things like arrays and nested objects if we need to add those.

Reply:
oh nice, I was unaware you could have code in your initializer like that.  That is neat!